.github/workflows/fuzz.yml

```diff
@@ -34,7 +34,7 @@ decode_program,
           key: cache-${{ matrix.target }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.58
+          toolchain: 1.64
```

Comment:
34075113e7873d4dfd15a2b1161b2ebbcffcabcc: fuzz/generate-files.sh needs to be updated.